Reference/Security/auto-linking/index.md

```diff
@@ -175,11 +175,131 @@ services.AddUmbraco(_env, _config)
 For some providers, it doesn't make sense to use auto-linking. This is especially true for public providers such as Google or Facebook. In those cases, it would mean that anyone who has a Google or Facebook account can log into your site. For public providers such as this, if auto-linking was needed you would need to limit the access by domain or other information provided in the Claims using the options/callbacks specified in those provider's authentication options.
 :::
 
+### Example for members
+The way to implement auto linking for members is fairly similar to how it is for users. The main difference is the UI, where Umbraco do not have a fixed login page for members.
+Instead, Umbraco ships with some Partial Macro Snippets for `Login` and `EditProfile` that contains handling of Login and manual linking of the configured external member providers.
+
+When auto-linking is enabled, only the `Login` snippet is relevant as users do not have to register before.
+
+The following example will show how to use Google and external login provider.
+You can first create a `GoogleMemberExternalLoginProviderOptions.cs` file which configures the options like
+
+
+```Csharp
+using Microsoft.Extensions.Options;
```

Comment:
Can you add a `using System;` please because of the following `Array.Empty<string>` on line 41